routers/api/v1/org/org.go

```diff
@@ -62,6 +62,56 @@ func ListUserOrgs(ctx *context.APIContext) {
 	listUserOrgs(ctx, u, false)
 }
 
+// Create api for create organization
```

Comment:
Can you add Gitea copyright to header of this file?

Reply:
Ugh sorry @techknowlogick forgot to look for that!

Reply:
@zeripath it's ok. It's something that is missed a lot, and we thankfully have git so if we do miss in one PR it we can just look at the history of the repo and add the appropriate dates when fixing.